uber/templates/registration/form.html

```diff
@@ -5,18 +5,19 @@
 {% include "registration/menu.html" %}
 
 <script type="text/javascript">
-    function isPreassignedBadgeType(badge_type) {
-        return $.inArray(parseInt(badge_type), [{{ STAFF_BADGE }}, {{ SUPPORTER_BADGE }}]) >= 0;
-    }
-    
-    function setGroupLink() {
-        var group = $.val("group_opt");
-        if( group == "" )
-            $("#group_link").hide();
-        else
-            $("#group_link").attr("href", "../groups/form?id=" + group).show();
-    }
-    
+    var isPreassignedBadgeType = function (badge_type) {
```

Comment:
Wait, I've never noticed this before. Shouldn't this be based on a config option?